xbmc/interfaces/Builtins.cpp

```diff
@@ -108,6 +109,7 @@
 using namespace std;
 using namespace XFILE;
 using namespace ADDON;
+using namespace PVR;
```

Comment:
to avoid the namespace usage we could use `PVR::CPVRManager::Get().TriggerSearchMissingChannelIcons()`